htdocs/core/class/html.formmail.class.php

```diff
@@ -2012,61 +2093,13 @@ public static function getAvailableSubstitKey($mode = 'formemail', $object = nul
 }
 
 
-require_once DOL_DOCUMENT_ROOT.'/core/class/commonobject.class.php';
-
 /**
- * Object of table llx_c_email_templates
+ * ModelMail
  *
- * TODO Move this class into a file cemailtemplate.class.php
+ * Object of table llx_c_email_templates
  */
-class ModelMail extends CommonObject
+class ModelMail
 {
-	/**
```

Comment:
Why removing code into ModelMail ?
May be your code was done on a too old branch. Try to :
- swith on your branch lamrani002:dropDownForModel
- merge branch origin/develop, fix conflict and repush